src/components/utility/CopyToClipboard/CopyToClipboard.tsx

```diff
@@ -101,6 +101,7 @@ const CopiedIcon = styled(CopiedSVG)<{
   background-color: ${({ copiedColor }) =>
     copiedColor ? (copiedColor as string) : colors.secondary.CasperGreen};
   border-radius: 0.125rem;
+  margin-left: ${pxToRem(5)};
```

Comment:
Generally speaking it's better to use flex now a days to space items like this out.  Or at the very least use margin right on the other element so the UI doesn't look to broken if this every wraps onto the next line

Reply:
@valfaro123 Does this make sense for you Victor?